### PR TITLE
[RI Ch03] NFVI Required State

### LINKED ISSUE #796: [RI MVP] NFVI required State
RI MVP work item for Snezka: NFVI Required State

Placeholder to track status, and ask Q&A, for the creation (finishing) of NFVI Required State documentation, including Manifest Validations. Write the content in the respective RI chapter(s).

Lead: @msidana (Manik), @opensource-tnbt (Sridhar)

### PR DESCRIPTION
**Pull request by msidana**
Fixes #796

### CODE REVIEW

**Comment by rabiabdel** [comment]
This will be discussed during TSC and merged if there was no objections.

**Comment by fuqiao123** [comment]
I would like to approve this commit. However seems I don't have that right. I can't submit review.

File: doc/ref_impl/cntt-ri/chapters/chapter03.md [hunk 1]
```diff
@@ -99,3 +100,110 @@ This profile is the global settings for the whole NFVI, which means there should
...
+| `req.sec.cmp.02` | **must** comply with all applicable standards and regulations. |
+| `req.sec.cmp.03` | **must** comply with all applicable regional standards and regulations. |
+| `req.sec.ntw.01` | **must** have the underlay network include strong access controls that comply with ISO 27001 and adhere to the V1.1 NIST Cybersecurity Framework. |
+| `req.sec.ntw.02` | **must** have all security logs stored in accordance with ISO27001. |
+| `req.sec.ntw.03` | **must** have the underlay network incorporate encrypted and/or private communications channels to ensure its security. |
+| `req.sec.ntw.04` | **must** configure all of the underlay network components to ensure the complete separation from the overlay customer deployments. |
+
+[RA1: Chapter 5 Interfaces and APIs](../../../ref_arch/openstack/chapters/chapter05.md) describes the baseline version regarding to Openstack Service APIs.
+
+| OpenStack Service | Link for API list | Baseline Version |
```
Comment (pgoyal01) [inline doc/ref_impl/cntt-ri/chapters/chapter03.md]: @msidana Please include micro-versions also -- see [5.3.1. OpenStack Interfaces](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md#531-openstack-interfaces). Thanks

**Comment by rabi-abdel** [comment]
@msidana can you please address @pgoyal01 comment below to get this merged

File: doc/ref_impl/cntt-ri/chapters/chapter03.md (hunk 1 above)
Comment (msidana) [reply doc/ref_impl/cntt-ri/chapters/chapter03.md]: Fixed.

**Comment by msidana** [comment]
> @msidana can you please address @pgoyal01 comment below to get this merged

Done.

**Comment by mf4716** [comment]
thank you Manik (@msidana ) - Team, one additional approval needed please in order to merge.

**Pull request merged**